### ISSUE TITLE
Fix(frontend): approve application takes time

### ISSUE THREAD

**Issue by kittybest** [privacy-scaling-explorations/maci-rpgf#213]
**Description**
After clicking approve, directly redirect back to the project page, but still see it's pending on approve, should either prevent user leave the approve process or show `loading...` instead of the pending on approve.

**Related Issues**
#93

**Comment by kittybest** [comment]
UPDATE

This is not due to submitting attestation onchain takes time, it actually happens immediately. So the problem is the `useQuery` is not updated even after refreshing the page, which means even it's refetching by calling `fetchAttestation` several times, always got same data, not until user head to other page and comeback to check the result.

So imagine that, if the coordinator approve on the `/applications` page, and after the `successfully!` popped up, click on the project and head to the `/projects/:projectId` page, would see this application be approved already.

**Comment by kittybest** [comment]
https://drive.google.com/file/d/12ePN2ShhnHEvsdwqiDeBK9YZ9uLUOK_N/view?usp=sharing

This is the test process of this bug.

**Comment by kittybest** [comment]
close by #218  and re-open another issue #229
